fullstack-cert/python-projects/binary-search-workshop/main.py

```diff
@@ -0,0 +1,27 @@
+def binary_search(search_list, value):
+    path_to_target = []
+    low = 0
+    high = len(search_list) - 1
+
+    while low <= high:
+        mid = (low + high) // 2
+
+        path_to_target.append(search_list[mid])
```

Comment:
```suggestion
        value_at_middle = search_list[mid]
        path_to_target.append(value_at_middle)
```
I guess this one is also a bit nitpicking, but it a tad cleaner, after changing  `search_list[mid]` in other places.